zyenyo-discord/src/main/java/asynchronous/typing/CompareTest.java

```diff
@@ -22,18 +24,26 @@ public class CompareTest implements Runnable
 	public CompareTest(MessageReceivedEvent event, String[] args)
 	{
 		this.event = event;
+		this.args = args;
 	}
 	
 	@Override
 	public void run()
 	{
+
 		try {
+			if (args.length == 2) {
+				idStr = args[1].subSequence(2, args[1].length()-1).toString();
+			} else {
+				idStr = event.getAuthor().getId();
+			}
+
```

Comment:
This is an older way of checking arguments, especially for pings. I think the newer for-cmd-switch statement in a parseArguments() method would be better for scalability, but it's fine for now, since I can't think of any other arguments we'd add to `\c`. Still good to keep in mind for later uses.

Also, there's a specific method to get a mentioned member within a message, `event.getMessage().getMentionedMembers().get(0).getId()`, which is better than manually checking the String, but the functionality is effectively the same.